components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -479,6 +484,6 @@ class PromptFeature(
     }
 
     companion object {
-        const val FILE_PICKER_REQUEST = 1234
+        const val FILE_PICKER_ACTIVITY_REQUEST_CODE = 1234
```

Comment:
This is outside of this PR, but it's probably better to allow the app to pass their own request code to avoid collisions with other request codes that may use the same values.

This may have been talked about before when initially landing and this was the best solution at the time.

Reply:
Yes, agreed. We need to do the same for activity request codes.